lms/djangoapps/courseware/tests/test_module_render.py

```diff
@@ -1103,7 +1103,7 @@ def test_static_badlink_rewrite(self):
         result_fragment = module.render(STUDENT_VIEW)
 
         self.assertIn(
-            '/c4x/{org}/{course}/asset/_file.jpg'.format(
+            '/c4x/{org}/{course}/asset/file.jpg'.format(
```

Comment:
Preliminary testing, on this branch, shows that creating a piece of static content with a filename of 'subdir/file.jpg' stores the content with an actual filename of 'subdir_file.jpg'.  This is both for split and mongo modulestores.

I haven't actually added an asset locally using Studio to see what it does.  Is specifying a subdirectory for uploading an asset even possible?